### DISCUSSION TITLE
Welcome to awesome-security-GRC Discussions!

### DISCUSSION THREAD

**Discussion by GRC-Engineer** [GRC-Engineer/awesome-security-GRC#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by veritysecurity** [comment]
Thanks for including me.
I'll start off with a question? How is Governance Risk and Compliance inter-related?
My simple answer is: An appropriate risk response is the objective, governance is the ability to manage the risk process and compliance is the measurement of risk. It is all about risk.

**Comment by ryanmrestivo** [comment]
Hello!  Thank you for including me as well.
I am a security consultant specializing in GRC and vulnerability management.
This awesome security GRC project is really exciting to me -- I hope to help by contributing knowledge and collaborating with our peers in the industry!
